Thanks for the new release, Greg.  I just tried it out on a system with
some disks, but a bunch of udev processes ended up hanging.  Is there
something I'm missing or do you need a patch like this?

Thanks,
Jesse

--- udev-004/udev-add.c	Mon Oct 20 14:39:08 2003
+++ udev-004-working/udev-add.c	Tue Oct 21 14:42:27 2003
@@ -141,7 +141,7 @@
 	strcat(filename, path);
 	strcat(filename, "/dev");
 
-	while (loop < SECONDS_TO_WAIT_FOR_DEV) {
+	while (loop++ < SECONDS_TO_WAIT_FOR_DEV) {
 		dbg("looking for %s", filename);
 		retval = stat(filename, &buf);
 		if (retval == 0) {


-------------------------------------------------------
This SF.net email is sponsored by OSDN developer relations
Here's your chance to show off your extensive product knowledge
We want to know what you know. Tell us and you have a chance to win $100
http://www.zoomerang.com/survey.zgi?HRPT1X3RYQNC5V4MLNSV3E54
_______________________________________________
Linux-hotplug-devel mailing list  http://linux-hotplug.sourceforge.net
Linux-hotplug-devel@lists.sourceforge.net
https://lists.sourceforge.net/lists/listinfo/linux-hotplug-devel